src/test/java/apoc/coll/CollTest.java

```diff
@@ -619,6 +619,45 @@ public void testFrequencies() throws Exception {
                 });
     }
 
+    @Test
+    public void testFrequenciesAsMapAsMap() throws Exception {
+        testCall(db, "RETURN apoc.coll.frequenciesAsMap([]) as value",
+                (row) -> assertEquals(Collections.emptyMap(), row.get("value")));
+        testCall(db, "RETURN apoc.coll.frequenciesAsMap(null) as value",
+                (row) -> assertEquals(Collections.emptyMap(), row.get("value")));
+        testCall(db, "RETURN apoc.coll.frequenciesAsMap([1,1]) as value",
+                (row) -> {
+                    Map<String, Object> maps= (Map<String, Object>) row.get("value");
+                    assertEquals(1, maps.size());
+                    assertEquals(2l, maps.get("1"));
```

Comment:
Mette la **L** di Long sempre maiuscola perché la **l** potrebbe confondersi in qualche caso con un 1
```suggestion
                    assertEquals(2L, maps.get("1"));
```